docs/config/examples/googlePubSub.md

```diff
@@ -0,0 +1,40 @@
+# Grafana Loki
+
+This example shows how to use `google-pubsub` as a destination for a `watch` data source.
+
+## Configuration
+
+Be sure to have read up on [Manzan configuration](/config/index.md) to understand where these files exist on your system.
+
+### `data.ini`
+
+```ini
+[watchout]
+type=watch
+id=sanjula
+destinations=pubsub_out
+format=$MESSAGE_ID$ (severity $SEVERITY$): $MESSAGE$
+strwch=WCHMSG((*ALL)) WCHMSGQ((*HSTLOG))
+```
+
+### `dests.ini`
+
+```ini
+[pubsub_out]
+type=google-pubsub
+
+# Set your project ID (ie. my-project-438217)
+projectId=<pubsub_project_id>
+
+# Set your topic name (ie. my-topic)
+topicName=<pubsub_topic_name>
+
+# Set your service account name (ie. file:/QOpenSys/etc/manzan/my-project-438217-b7392819a7hf.json)
+serviceAccountKey=<pubsub_service_account_key>
+```
+
+## Result
+
+<div style="text-align: center; margin: 20px;">
+    <img src="https://github.com/ThePrez/Manzan/blob/main/docs/images/googlePubSub1.png?raw=true" alt="Google Pub/Sub 1" style="box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2); border-radius: 8px; max-width: 100%; display: block; margin-bottom: 20px;">
```

Comment:
Can we specify the relative path here instead? We can put the images in /docs/assets/images

Reply:
Originally that was the plan, but I noticed that when you run the docs locally using `docsify serve docs` the images appear correctly with relative paths but on the live docs site it does not. Seems to only be an issue when you use `img` tags. Using img tags to add some nice formatting

You can see the issue on the Grafana Loki page on the live docs site: https://theprez.github.io/Manzan/#/config/examples/grafanaLoki

Reply:
That's because the path is wrong. It needs to be in /docs/assets/images, not /docs/images 
![image](https://github.com/user-attachments/assets/605ae0c8-dbf2-4b75-9450-105c02152a62)